travel_platform/izi_travel_client.py

```diff
@@ -10,15 +10,18 @@ class IziTravelApiError(BaseExceptionHandler):
     pass
 
 
-class IziTravelClient(object):
+class BaseIziTravelApiClient(object):
 
-    def __init__(self):
-        self.api_key = "9ba66b8d-2bdc-4674-bdc4-3cc67e2c25af"
-        self.api_url = "https://api.izi.travel/"
+    API_URL = "https://api.izi.travel"
```

Comment:
It may be, but I highly recommend to use configuration files for storing configurations (not Python modules).

It may be something like this:

```configs = read_from_configs(<configuration_file>)```

Or something similar.
It's much expected to find configurations in configuration file, than in some Python module. Very often such modules uses for cases when some additional calculation is needed.